### PR TITLE
Added bread flour and brown bread to more recipes

### PR DESCRIPTION
**Pull request by TheDarkMaster13**
#### Summary
Balance "Add bread flour as an alternative to flour to most bread recipes and allowed brown bread to be used for sandwiches"

#### Purpose of change
The existing bread flour item is only used to make brown bread and said brown bread cannot be used for anything.  This change makes it so that bread flour can be used for most bread recipes and brown bread can be used for sandwiches.

#### Describe the solution
Bread flour can now be used in place of flour in many bread recipes, including waffles and pancakes.  The bread flour recipe was also altered to use the same recipe times as normal quern made flour and so it produces the same amount of flour overall.

#### Testing
Tested on most recent experimental. Game ran and recipes all show the correct substitution options.  Bread, flatbread, wastebread, and others can now be made with bread flour.  Brown bread is now a valid option for sandwiches.

### CODE REVIEW

**Comment by wapcaplet** [comment]
Thanks for working on this. It will be nice to have better bread options.

It appears you used the same branch (your fork's `master`) for both this and your other PR #53381. Luckily since this is your only open pull request at the moment, and all your previous changes were already merged into master, it may still merge cleanly. However I would suggest creating a new branch in your fork for each future pull request - this keeps the changes from each PR isolated (and allows you to keep your master branch unpolluted, and in sync with the upstream master branch).

You could perhaps still tidy it up and move these changes to a new branch per [instructions like this](https://www.riccardomariabianchi.com/git-ops-i-forgot-to-create-a-branch-collecting-commits-to-separate-branch.html) but I think you would need to open a new PR in that case.

**Comment by zlsalmon** [comment]
Worth mentioning that the food referred to as "brown bread" in CDDA is a [sweet, dense product](https://www.today.com/food/what-canned-bread-nostalgic-new-england-treat-making-comeback-t182506) resembling pound cake and packaged in cans; I wouldn't necessarily think of it as an appropriate medium for sandwiches and the like, but then I would have said that about crackers too.

**Comment by wapcaplet** [comment]
> Worth mentioning that the food referred to as "brown bread" in CDDA is a [sweet, dense product](https://www.today.com/food/what-canned-bread-nostalgic-new-england-treat-making-comeback-t182506) resembling pound cake and packaged in cans; I wouldn't necessarily think of it as an appropriate medium for sandwiches and the like, but then I would have said that about crackers too.

This came up on the development discord too, a product I was not aware of when I told @TheDarkMaster13 that it seemed like a good sandwich bread based on the ingredience. I see now that the original PR that added it #45576 does indeed refer to the canned bread product - it's unfortunate that the item description says nothing about that.

File: data/json/requirements/cooking_components.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
         [ "sourdough_bread", 1 ],
         [ "biscuit", 1 ],
         [ "brioche", 1 ],
-        [ "crackers", 1 ]
+        [ "crackers", 1 ],
+        [ "brown_bread", 1 ]
       ]
```
Comment (ZhilkinSerg) [inline data/json/requirements/cooking_components.json:35]:
```suggestion
       ]
```

File: data/json/requirements/cooking_components.json [hunk 2]
```diff
@@ -30,8 +30,9 @@
         [ "sourdough_bread", 1 ],
         [ "biscuit", 1 ],
         [ "brioche", 1 ],
-        [ "crackers", 1 ]
-      ]
+        [ "crackers", 1 ],
+        [ "brown_bread", 1 ]
+       ]
```
Comment (ZhilkinSerg) [inline data/json/requirements/cooking_components.json:35]:
```suggestion
      ]
```

**Pull request merged**